### ISSUE TITLE
The response value should be typed. 

### ISSUE THREAD

**Issue by dwjohnston** [jxom/react-loads#21]
ie. if my async function has this signature: 

```
export const doGetThingys = async (): Promise<Thingys> => {
    //implementation
}

//...elsewhere
const getThingys = React.useCallback(doGetThingys,[]); 
const { response, error} = useLoads(getThingys);
```

Then when I use `useLoads` - I should have be getting a `Thingys` typed object as the response, not `any`.

**Comment by jxom** [comment]
Valid point - I'll have a look at this tonight.

**Comment by jxom** [comment]
Fixed in `8.1.0`. Let me know how you go.
